rustfmt.toml

```diff
@@ -1,2 +1,2 @@
 write_mode = "Overwrite"
-
+closure_block_indent_threshold = 1
```

Comment:
Ahh, this is why some of the changes are looking so odd to me. Why do you want this here?